specifications/transactions/travel-rule/README.md

```diff
@@ -0,0 +1,372 @@
+				},
+			    	"status": {
+			    		"status": "ready_for_settlement",
+			    	}
+			},
+		    "receiver": {
+			    "address": "dm1pgfpyysjzgfpyysjzgfpyysjzgf3xycnzvf3xycsm957ne",
+			},
+		    "reference_id": "5b8403c9-86f5-3fe0-7230-1fe950d030cb",
+		    "action": {
+			    "amount": 100,
+			    "currency": "USD",
+			    "action": "charge",
+			    "timestamp": 72322,
+			},
+		    "description": "A free form or structured description of the payment.",
+		},
+	},
+}
+```
+
+A successful response to this command should look like the following:
+
+```
+{
+    "_ObjectType": "CommandResponseObject",
+    "status": "success",
+}
+```
+
+A travel-rule exchange may require multiple `PaymentCommand`'s in order to complete. In these instances, the underlying structures may be passed back and forth many times. As outlined in the rest of this section, certain fields must be defined at the outset and others can be defined later. Some fields may be adjusted or mutated throughout the exchange, whereas others must remain immutable.
+
+### PaymentCommand Object
+
+| Field          | Type            | Required | Description |
+|-------         |------           |----------|-------------|
+| `_ObjectType`  | `str`           | Y        | The fixed string `PaymentCommand` |
+| `payment`      | `PaymentObject` | Y        | Contains a `PaymentObject` |
+
+```
+{
+    "_ObjectType": "PaymentCommand",
+    "payment": {
+        PaymentObject(),
+    }
+}
+```
+
+### PaymentObject
+
+Some fields are immutable after they are defined once. Others can be updated multiple times (see below). Updating immutable fields with a different value results in a Command error.
+
+| Field                           | Type                  | Required | Description |
+|-------                          |------                 |----------|-------------|
+| `sender`/`receiver`             | `PaymentActorObject`  | Y        | Information about the sender/receiver in this payment |
+|`reference_id`                   | `str`                 | Y        | Unique reference ID of this payment on the payment initiator VASP (the VASP which originally created this payment Object). This value should be globally unique. This field is mandatory on payment creation and immutable after that. This must be a UUID according to RFC4122 with "-"s included. |
+| `original_payment_reference_id` | `str`                 | N        | Used to refer an old payment known to the other VASP. For example, used for refunds. The reference ID of the original payment will be placed into this field. This field is mandatory on refund and immutable |
+| `recipient_signature`           | `str`                 | N        | Signature of the recipient of this transaction encoded in hex. The is signed with the compliance key of the recipient VASP and is used for on-chain attestation from the recipient party. This may be omitted on blockchains which do not require on-chain attestation. Generated via Recipient Signature |
```

Comment:
Is there more details on what exactly is being signed by the recipient?

Reply:
This is a good point, we really should probably either include the spec of what is being signed or provide a link to it here.

Reply:
yup, we need a guide / spec that connects all these things together